setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = readthedocs
-version = 2.2.1
+version = build
```

Comment:
What is the implication in changing this?

Reply:
I didn't changed it.. It got automatically changed when i ran <code>invoke prepare build</code> ... Looks like I had to pass version number in place of build.